src/components/Card.js

```diff
@@ -15,43 +20,60 @@ class Card {
   }
   
   _setEventListeners() {
-    this._element
-      .querySelector(".card__like-button")
-      .addEventListener("click", () => this._handleLikeButton());
+    
+    this._cardImageEl.addEventListener("click", () => {
+      this._handlePreviewPicture(this._name, this._link)
+    });
+    
+    this._deleteButton = this._element.querySelector(".card__delete-button");

```

Comment:
`    this._deleteButton = this._element.querySelector(".card__delete-button");
`

You've found `this._deleteButton` in line 69. No need to search it again